.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 
 before_install:
+  - npm i -g npm
```

Comment:
I don't like doing this - as npm has been known to break itself even with minor or patch releases. Since the .travis.yml file should be supplying Node versions, it also reliably gets the npm version bundled with Node

I think this was added as a troubleshooting step. If so, please remove.

Reply:
We focused on Lerna for our initial monorepo conversion. I don't want to get into the game of documenting everything with npm and yarn. npm 5+ has come a long way., blunting the original benefits of yarn to me.  We do not use lockfiles (as you can see the problems in this PR) anyways, further blunting the need. I understand we could use yarn for dev and posture to users only npm, but I don't really understand the need for yet another tool.  Please let me know if I am missing something here 😅

Reply:
Less tools is better IMO :)

Reply:
Was done as a troubleshooting step; removed now.